src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help-manifestPath.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Path of deployit-manifest.xml relative to workspace. Jenkin's environment variables can be used to specify values inside manifest file.
```

Comment:
Path of the deployit-manifest.xml file, relative to the workspace. Jenkin's environment variables can be used to specify values in the manifest file.